include/sys/math.h

```diff
@@ -98,7 +98,7 @@ f32 fsin16(s16 theta);
 
 f32 sinf(f32);
 
-s16 arctan2s(s16 x, s16 z);
+s16 arctan2s(s32 x, s32 z);
```

Comment:
Are you sure these are s32? arctan2s is unfortunately handwritten asm, not sure how to verify this